tests/wpt/web-platform-tests/html/semantics/document-metadata/the-link-element/link-style-error-01.html

```diff
@@ -7,17 +7,32 @@
 <div id="log"></div>
 <div id="test">
 <script>
-//var t404 = async_test("Should get an error event for a 404 error.")
-//t404.step(function() {
-//  var elt = document.createElement("link");
-//  elt.onerror = t404.step_func(function() {
-//    assert_true(true, "Got error event for 404 error.")
-//    t404.done()
-//  })
-//  elt.rel = "stylesheet";
-//  elt.href = 404 error;
-//  document.getElementsByTagName("head")[0].appendChild(elt);
-//})
+var t404 = async_test("Should get an error event for a 404 error.")
+t404.step(function() {
+  var elt = document.createElement("link");
+  elt.onerror = t404.step_func(function() {
+    assert_true(true, "Got error event for 404 error.")
+    t404.done()
+  })
+  elt.onload = t404.unreached_func("load event should not be fired");
```

Comment:
This will not catch a load event dispatched after the error event, because the test is done by then.